provider/parameter_test.go

```diff
@@ -424,6 +440,54 @@ data "coder_parameter" "region" {
 				require.Equal(t, expected, state.Primary.Attributes[key])
 			}
 		},
+	}, {
+		Name: "NumberValidation_MonotonicWithOptions",
+		Config: `
+			data "coder_parameter" "region" {
+			  name        = "Region"
+              type        = "number"
```

Comment:
nit: formatting?

Reply:
Oh weird, my editor is probably handling tabs in a strange way. Thanks 👍